src/h264_subscriber.cpp

```diff
@@ -37,7 +38,7 @@ void H264Subscriber::subscribeImpl (ros::NodeHandle &nh, const std::string &base
     SimpleSubscriberPlugin::subscribeImpl(nh, base_topic, queue_size, callback, tracked_object, transport_hints);
 }
 
-void H264Subscriber::internalCallback(const h264_image_transport_msgs::H264Packet::ConstPtr& message, const Callback& user_cb)
+void H264Subscriber::internalCallback(const sensor_msgs::CompressedImage::ConstPtr& message, const Callback& user_cb)
 {
     avpkt.size = message->data.size();
     avpkt.data = (uint8_t*)&message->data[0];
```

Comment:
Isn't `&message->data[0]` equivalent to `message->data`?

Reply:
This would be true for a C array. Here `message->data` is a std::vector.
But, in any case, to access a vector's pointer,  [`std::vector<T,Allocator>::data`](https://en.cppreference.com/w/cpp/container/vector/data) should be used. 
Like so: `message->data.data()`